packages/expo-web-browser/src/WebBrowser.ts

```diff
@@ -149,7 +149,18 @@ let _redirectHandler: ((event: RedirectEvent) => void) | null = null;
 // returns to active
 let _onWebBrowserCloseAndroid: null | (() => void) = null;
 
+// Store previous app state to check whether the listener has ever been attached
+let _previousAppState: null | string = AppState.currentState;
```

Comment:
It'll be better if we change it to `bool` and rename it. We don't have to remember the previous app state. We need information if it the first call or not.